### PR TITLE
Changes to how typings are exported

### PR DESCRIPTION
**Pull request by jpbberry**
Pretty sure this is how I should've done it in the first place.

Allows for easier importing and usage of types within external packages.

### CODE REVIEW

File: src/structs/Api.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import ApiError from '../utils/ApiError'
 import qs from 'querystring'
 import { EventEmitter } from 'events'
 
+import { Snowflake, BotStats, BotInfo, UserInfo, BotsResponse, ShortUser, BotsQuery } from '../typings'
+
```
Comment (Xetera) [inline src/structs/Api.ts:7]: Are these unexported types still accessible from `./dist/typing.d.ts`?

Comment (jpbberry) [reply src/structs/Api.ts:7]: All of the types are exported in typings.ts

Comment (jpbberry) [reply src/structs/Api.ts:7]: So yes you can do
```ts
import { BotStats } from '@top-gg/sdk/dist/typings'
```
In external projects

Comment (Xetera) [reply src/structs/Api.ts:7]: oh I see what the issue was then ok

**Review by Xetera — APPROVED** [review]

**Pull request merged**